client/my-sites/plans/jetpack-plans/category-filter/index.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+	JETPACK_SECURITY_CATEGORY,
+	JETPACK_PERFORMANCE_CATEGORY,
+	JETPACK_GROWTH_CATEGORY,
+	JETPACK_PRODUCT_CATEGORIES,
+} from '@automattic/calypso-products';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { useMemo, useState, useCallback, useEffect } from 'react';
+import CategoryFilterItem from './item';
+import type { JetpackProductCategory } from '@automattic/calypso-products';
+
+import './style.scss';
+
+type Props = {
+	className?: string;
+	defaultValue?: JetpackProductCategory;
+	onChange: ( category: JetpackProductCategory ) => void;
+};
+
+const CategoryFilter: React.FC< Props > = ( { className, defaultValue, onChange } ) => {
+	const [ value, setValue ] = useState< JetpackProductCategory | undefined >( defaultValue );
+	const translate = useTranslate();
+	const translations = useMemo(
+		() => ( {
+			[ JETPACK_SECURITY_CATEGORY ]: translate( 'Security' ),
+			[ JETPACK_PERFORMANCE_CATEGORY ]: translate( 'Performance' ),
+			[ JETPACK_GROWTH_CATEGORY ]: translate( 'Growth' ),
+		} ),
+		[ translate ]
+	);
+	const targets = useMemo(
+		() => ( {
+			[ JETPACK_SECURITY_CATEGORY ]: translate( 'security' ) as string,
+			[ JETPACK_PERFORMANCE_CATEGORY ]: translate( 'performance' ) as string,
+			[ JETPACK_GROWTH_CATEGORY ]: translate( 'growth' ) as string,
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **36** times:
`translate( 'Growth' )` *ES Score: 13*